mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -207,6 +207,13 @@ examples:
       # Ignore this field as it is INPUT_ONLY. AR will not return this in the
       # response.
       - 'remote_repository_config.0.disable_upstream_validation'
+  - name: 'artifact_registry_repository_vulnerability_scanning'
+    primary_resource_id: 'my-repo'
+    primary_resource_name: 'fmt.Sprintf("tf-test-my-repository%s", context["random_suffix"])'
+    vars:
+      repository_id: 'my-repository'
+      desc: 'example docker repository with vulnerability scanning config'
+      enablement_config: 'INHERITED'
```

Comment:
This should fix this test's failures by ensuring this uses a literal value - or you could hardcode it in the test template (since it's not likely to be reused or modified.)

```suggestion
      enablement_config: 'INHERITED'
    test_vars_overrides:
      enablement_config: 'INHERITED'
```

Reply:
Moved the field inline in the example.